confidential-data-hub/hub/protos/api.proto

```diff
@@ -29,9 +29,7 @@ message SecureMountRequest {
     string mount_point = 4;
 }
 
-message SecureMountResponse {
-    string mount_path = 1;
-}
+message SecureMountResponse {}
```

Comment:
I can maybe imagine cases where the exact mount point isn't known until mount time and the CDH would want to report this to the caller, but we can worry about that later I guess.